cmd/sandbox-ce/cmd/config.go

```diff
@@ -0,0 +1,201 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/mitchellh/go-homedir"
```

Comment:
we don't need external package for it anymore. Go 1.12 introduced function to get home dir in stdlib. Take a look at: https://github.com/src-d/superset-compose/pull/51

Reply:
done